src/cmake/module/add_external_resources.cmake

```diff
@@ -135,16 +72,21 @@ macro(add_external_resources target)
       foreach(resource ${resources})
         get_filename_component(filename "${resource}" NAME)
         set(output_file "${build_dir}/${filename}")
-        add_custom_command(
-          OUTPUT "${output_file}"
+        add_custom_command(TARGET ${target} POST_BUILD
+          BYPRODUCTS "${output_file}"
           COMMAND ${CMAKE_COMMAND} ARGS -E copy "${resource}" "${output_file}"
           )
-        string(REPLACE " " "_" target_suffix "${filename}")
-        add_custom_target(${target}_resource_${target_suffix} ALL
-          DEPENDS "${resource}" "${output_file}")
+        list(APPEND copied_resources "${output_file}")
       endforeach()
 
+      set(resources "${copied_resources}")
+
     endif()
+
```

Comment:
The lines below have not changed. They are just moved here for clarity (the above code handles build rules, the code below handles packaging, so it is more logical in this order).